portfolio/src/main/webapp/index.html

```diff
@@ -1,18 +1,270 @@
+			<div class="card-body">
+				<h5 class="card-title">Learner and Explorer</h5>
+				<p class="card-text">I solve problems. I yearn to create and implement Computer Science solutions. It
+					really interests me how Computer
+					Science can have such a big impact in our daily lives. I wish to use my software design, analysis
+					and programming skills to create innovavtive, automated
+					systems. I love implement my knowledge in various hackathons. I also like particiating in elecutions
+					and MUNs.
+					I am a hardworking, confident and consistend performer who does not give up on my goals easily.
+				</p>
+
+			</div>
+		</div>
+	</div>
+	<!-- About me end -->
+
+
+	<!-- jumbotron2 start -->
+	<div class="jumbotron myjumbo" id="achievements">
+		<h1 class="display-4">Achievements</h1>
+
+		<hr class="my-4">
+		<div class="shadow p-3 mb-5 bg-white rounded achievementbar">Awarded Mrs. Phool Devi scholarship by Hon'ble
+			Education Minister for
+			excellent academic performance in IGDTUW</div>
+		<div class="shadow p-3 mb-5 bg-white rounded achievementbar">CodeChef May Luchtime : Global Rank 476 (2019).
+		</div>
+		<div class="shadow p-3 mb-5 bg-white rounded achievementbar">Winner of Dare To Manage ( Coding Contest ) during
+			Innerve ( Technical
+			Fest, IGDTUW)
+		</div>
+		<div class="shadow p-3 mb-5 bg-white rounded achievementbar">Winner of Know The Unkown ( Coding Contest) during
+			Innerve ( Technical
+			Fest, IGDTUW)
+		</div>
+
+	</div>
+	<!-- jumbotron2 end -->
+
+	<div class="myjumbo" id="community">
+		<h1>Community</h1>
+		<div class="card mb-3 lowecard">
+			<img src="images/aluminimeet.jpg" class="card-img-top" alt="...">
+			<div class="card-body">
+				<h5 class="card-title">I love interacting with people and getting to know more about them</h5>
+				<p class="card-text">Meeting people and building connect is a quality that every individual should have.
+					I believe in rising with community.
+					I Received Certificate of “Special Appreciation of Efforts” from HelpAge India for creating
+					awareness and assisting in raising funds for the care of elderly. I believe in using my skills to
+					help the community.
+					I had made a chat bot called “Companion Bot” for Dementia patients. Dementia patients often forget
+					their basic information and due to this disease, they lack social interaction in their lives. The
+					badly need a
+					friend so that they can communicate without any hesitation. The bot is a companion for dementia
+					patients. The patient can ask for his basic information from the bot again and again, talk to the
+					bot about his day, play songs, watch movies by links provided with the bot, etc. So this was a
+					project implemented to help dementia patients.
+				</p>
+
+			</div>
+		</div>
+	</div>
+
+
+	<!-- jumbotron3 start -->
+	<div class="jumbotron myjumbo" id="skills">
+		<h1 class="display-4">Skills</h1>
```

Comment:
'Skills' and 'Project' headings are styled differently if it's intentional then it's fine

Reply:
yes